libcomposefs/lcfs-writer.c

```diff
@@ -606,6 +606,62 @@ static void digest_to_path(const uint8_t *csum, char *buf)
 	buf[j] = '\0';
 }
 
+bool lcfs_node_set_from_content(struct lcfs_node_s *node, int dirfd,
+				const char *fname, int buildflags)
```

Comment:
All other functions return int, where 0 == success and < 0 is error. I think we should keep doing that.

Reply:
fixed